tensorflow_privacy/privacy/membership_inference_attack/models.py

```diff
@@ -110,6 +112,93 @@ def _column_stack(logits, loss):
   return np.column_stack((logits, loss))
 
 
+def create_seq2seq_attacker_data(attack_input_data: Seq2SeqAttackInputData,
+                                 test_fraction: float = 0.25,
+                                 balance: bool = True) -> AttackerData:
+  """Prepare Seq2SeqAttackInputData to train ML attackers.
+
+  Uses logits and losses to generate ranks and performs a random train-test split.
+
+  Args:
+    attack_input_data: Original Seq2SeqAttackInputData
+    test_fraction: Fraction of the dataset to include in the test split.
+    balance: Whether the training and test sets for the membership inference
+              attacker should have a balanced (roughly equal) number of samples
+              from the training and test sets used to develop the model
+              under attack.
+
+  Returns:
+    AttackerData.
+  """
+  attack_input_train = _get_average_ranks(attack_input_data.logits_train, attack_input_data.labels_train)
+  attack_input_test = _get_average_ranks(attack_input_data.logits_test, attack_input_data.labels_test)
+
+  if balance:
+    min_size = min(attack_input_data.train_size,
```

Comment:
It looks the only place where train/test_size is needed. But it the same as the number of elements attack_input_train. So we don't need train/test_size.
If that's true, please drop train/test_size from attack_input_data (no need to ask a API user to specify something that it's possible to figure out.

Reply:
I've used train_size and test_size to print out the attack input shape in the following [part](https://github.com/tensorflow/privacy/pull/137/files#diff-45da7eb7a63fd5e4edabfc2f14124a5e1cde7cffdaf242d424838484de0bff04R439). Since logits and labels are being lazy loaded the user needs to specify train_size and test_size. 

But we could drop train_size and test_size if we don't need to print out the exact attack input shape?